docs/area-owners.md

```diff
@@ -17,16 +17,14 @@ The **dotnet/runtime** repo has [its own list](https://github.com/dotnet/runtime
 | **[area-infrastructure](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-infrastructure)**     | [@wtgodbe](https://github.com/wtgodbe)                                                        | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           |                         |
 | **[area-middleware](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-middleware)**             | [@mitchdenny](https://github.com/mitchdenny)                                                  | [@adityamandaleeka](https://github.com/adityamandaleeka) |                         |
 | **[area-minimal](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-minimal)**                   | [@captainsafia](https://github.com/captainsafia) [@mitchdenny](https://github.com/mitchdenny) | [@adityamandaleeka](https://github.com/adityamandaleeka) |                         |
-| **[area-mvc](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-mvc)**                           | [@captainsafia](https://github.com/captainsafia)                                              | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           | @mkArtakMSFT owns views, @adityamandaleeka owns the other parts                        |
+| **[area-mvc](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-mvc)**                           | [@captainsafia](https://github.com/captainsafia)                                              | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           | @mkArtakMSFT owns views, @adityamandaleeka owns the other parts |
```

Comment:
I think we can update this now since we decide that `area-ui-rendering` was exclusively for views and `area-mvc` was for everything else?